Android.mk

```diff
@@ -45,11 +45,19 @@ HYBRIS_FIXUP_MOUNTS := $(LOCAL_PATH)/fixup-mountpoints
 # TARGET_VENDOR := $(shell echo $(PRODUCT_MANUFACTURER) | tr '[:upper:]' '[:lower:]')
 # but Cyanogenmod seems to use device/*/$(TARGET_DEVICE) in config.mk so we will too.
 HYBRIS_FSTABS := $(shell find device/*/$(TARGET_DEVICE) -name *fstab* | grep -v goldfish)
+ifeq "$(HYBRIS_FSTABS)" ""
+TARGET_VENDOR := "$(shell echo $(PRODUCT_MANUFACTURER) | tr '[:upper:]' '[:lower:]')"
```

Comment:
This is quite involved case, so a comment would be helpful. Since it might affect AOSP and/or ia (Intel Architecture) variants